NitroxServer-Subnautica/GameLogic/Entities/Spawning/SubnauticaEntitySpawnPointFactory.cs

```diff
@@ -8,36 +9,64 @@ namespace NitroxServer_Subnautica.GameLogic.Entities.Spawning
 {
     public class SubnauticaEntitySpawnPointFactory : EntitySpawnPointFactory
```

Comment:
Of course ESPs can have children.... UGGG UWE haha.

Reply:
Yea i know right?